app/src/main/java/com/dmonzonis/nookpendium/CollectionActivity.kt

```diff
@@ -42,16 +56,45 @@ class CollectionActivity : AppCompatActivity() {
 
             override fun onTabSelected(tab: TabLayout.Tab?) {
                 // TODO: Depending on hemisphere (set in settings), get the NH or SH xml file for ACNH
-                val filename = when (tab?.text.toString()) {
-                    getString(R.string.fish) -> getString(R.string.fish_file)
-                    getString(R.string.bugs) -> getString(R.string.bugs_file)
-                    else -> getString(R.string.underwater_file)
-                }
-                val inputStream: InputStream = assets.open(filename)
-                val recordList: List<Record> = parser.parse(inputStream)
-                viewAdapter = RecordListAdapter(recordList)
-                recyclerView.adapter = viewAdapter
+                loadGameAssets(tab)
             }
         })
     }
+
+    override fun onOptionsItemSelected(item: MenuItem?): Boolean {
+        if (drawerToggle.onOptionsItemSelected(item)) {
+            return true
+        }
+        return super.onOptionsItemSelected(item)
+    }
+
+    private fun loadGameAssets(tab: TabLayout.Tab?) {
+        val filePath = if (selectedGame == R.string.game_acnh) {
+            if (tabLayout.tabCount > 2) {
+                tabLayout.removeTabAt(2)
```

Comment:
Maybe put this logic outside of the variable assignation to make it a bit easier to read, same with the one in the `else`